src/main/java/org/jenkinsci/tools/configcloner/Main.java

```diff
@@ -47,7 +47,10 @@
     public static void main(final String[] args) {
 
         final CommandResponse resp = CommandResponse.system();
-        final CLIPool cliPool = new CLIPool(CLIFactory.system());
+        String sshKeyLocation = System.getenv("SSH_PRIVATE_KEY");
+        final CLIPool cliPool = new CLIPool(sshKeyLocation == null ?
+                CLIFactory.system() :
+                CLIFactory.provided(new String[]{sshKeyLocation}));
```

Comment:
No need to wrap in `new String[]{...}`, the argument is variadic.

Reply:
Thanks, updated